src/content/structured/community/roadmap.mdx

```diff
@@ -24,27 +24,52 @@ Check out our <ic-link target="_blank" href="https://github.com/orgs/mi6/project
 
 ## Recently shipped
 
-- Release of [skip link component](/components/skip-link).
-- Canary release of time input component.
-- Release of action chip component.
-- Release of [layout grid component](/components/layout-grid).
-- Canary release of [table of contents component](/components/table-of-contents).
-- Updated code examples on website.
-- Tokens and theming of components for dark mode.
+### Components
 
-For a full list of recent changes made, please read the <ic-link target="_blank" href="https://github.com/mi6/ic-ui-kit/blob/main/packages/web-components/CHANGELOG.md" rel="noreferrer noopener">changelog</ic-link>
+- Optional CSS theme for native tables.
+- Addition of cursor position to [IcTextField](/components/text-field) key down event.
+- An update to [IcToggleButton](/components/toggle-button) to reflect IcToggleButtonGroup disabled state.
+- Fix to select all rows in [IcDataTable](/components/data-table).
+- Improvements to [IcTextField](/components/text-field) validation aria-live behaviour, addition of validationAriaLive prop.
+- Addition of ability to hide [IcSelect](/components/select) menu option labels.
+
+For a full list of recent component changes made, please read the <ic-link target="_blank" href="https://github.com/mi6/ic-ui-kit/blob/main/packages/web-components/CHANGELOG.md" rel="noreferrer noopener">changelog</ic-link>
+
+### Guidance website
+
+- Update component gallery images to ensure consistent typography.
```

Comment:
I'm wondering if this should just be:
```suggestion
- Update component gallery images.
```
because we're also adding the images that are missing for some components, so this would cover those as well. (I think the designers might have changed other things, not just typography, in some images too)